src/common/util/include/openvino/util/xml_parse_utils.hpp

```diff
@@ -33,27 +28,23 @@ IE_SUPPRESS_DEPRECATED_START
  */
 #define FOREACH_CHILD(c, p, tag) for (auto c = p.child(tag); !c.empty(); c = c.next_sibling(tag))
 
-/**
- * @brief XML helpers function to extract values from `pugi::xml_node`
- * @ingroup    ie_dev_api_xml
- */
-namespace pugixml {
+namespace ov {
+namespace util {
 
 /**
  * @brief XML helpers function to extract values from `pugi::xml_node`
- * @ingroup    ie_dev_api_xml
  */
+namespace pugixml {
 namespace utils {
```

Comment:
do we need this inner `utils`?
`utils` inside `ov::util` looks redundant

Reply:
yes, I think so, just only looks similar origin namespace. Ok, will remove inner utils.

Reply:
done